packages/makeswift/src/integrate-next-app.ts

```diff
@@ -232,24 +244,80 @@ export default function Page({ snapshot }: Props) {
 
   // custom document
   if (
-    fs.existsSync(path.join(pagesFolder, `_document.${extension}`)) ||
-    fs.existsSync(path.join(pagesFolder, `_document.${extension}x`))
+    fs.existsSync(path.join(pagesFolder.absolute, `_document.${extension}`)) ||
+    fs.existsSync(path.join(pagesFolder.absolute, `_document.${extension}x`))
   ) {
     throw new MakeswiftError(
       'A custom document already exists, you will have to manually integrate: https://www.makeswift.com/docs/guides/manual-setup#set-up-custom-document',
```

Comment:
We probably want to link to this instead: https://www.makeswift.com/docs/guides/advanced-setup#extending-the-custom-document

The link that's currently there is not very helpful as it's just a 1 liner export that doesn't explain how to use a custom document together with Makeswift